emf/common/xslt_engine/saxonpy_api.py

```diff
@@ -104,7 +103,7 @@ def validate_xml(input_xml, schema_xml):
     with open('QAR_v2.6.1.xsd', 'rb') as file:
         xsd_bytes = file.read()
 
-    data = {"XML": xml_bytes.decode(),"XSL": xsl_bytes.decode(), "XSD": xsd_bytes.decode()}
+    data = {"XML": xml_bytes.decode(), "XSL": xsl_bytes.decode(),}#"XSD": xsd_bytes.decode()}
```

Comment:
Leave here working example, the referenced transformation is not present, also XSD should be under schemas folder